Docs - No frontend multiple columns search warning added (#12508)
supabase/supabase: apps/docs/pages/guides/database/full-text-search.mdx

### Search multiple columns

> To find all `books` where `description` or `title` contain the word `little` (there is no JavaScript and Dart frontend api for multiple columns search at this moment):

Reviewer: > there is no JavaScript and Dart frontend api for multiple columns search at this moment

Note that this is not true because computed columns are part of the API: https://postgrest.org/en/stable/api.html#computed-virtual-columns. They're explicitly designed for this type of use case.

> To find all `books` where `description` or `title` contain the word `little` (there is no JavaScript and Dart frontend api for multiple columns search at this moment):

Reviewer 2: ```suggestion
To find all `books` where `description` or `title` contain the word `little` (you can used [computed columns](https://postgrest.org/en/stable/api.html#computed-virtual-columns) to concatenate multiple columns):
```
